paddle/fluid/operators/roi_pool_op.cu

```diff
@@ -169,12 +192,30 @@ class GPUROIPoolGradOpKernel : public framework::OpKernel<T> {
     auto pooled_width = ctx.Attr<int>("pooled_width");
     auto spatial_scale = ctx.Attr<float>("spatial_scale");
 
-    size_t rois_num = rois->dims()[0];
+    int rois_num = rois->dims()[0];
     int channels = in->dims()[1];
     int height = in->dims()[2];
     int width = in->dims()[3];
 
     if (x_grad) {
+      framework::Tensor roi_batch_id_list;
+      framework::DDim roi_batch_id_shape = {rois_num};
+      roi_batch_id_list.Resize(roi_batch_id_shape);
+      int* roi_batch_id_data =
+          roi_batch_id_list.mutable_data<int>(platform::CPUPlace());
+      auto rois_lod = rois->lod();
+      int rois_batch_size = rois_lod[0].size() - 1;
+      auto rois_index = rois_lod[0];
```

Comment:
Same as above, the code can be written simply.

Reply:
thanks, this format is simpler.